lib/test.js

```diff
@@ -239,31 +233,45 @@ Test.prototype._publicApi = function () {
 		self._assert();
 	}
 
+	function onAssertionEvent(event) {
+		if (event.type === 'error') {
```

Comment:
@vdemedes - This event now happens for every assertion. Pass or fail.

We are changing `event.type === 'error' / 'success'` to `event.assertionThrew === true/false`

see https://github.com/twada/empower-core/pull/2 for further discussion.

`event.originalMessage` will contain the original message the user passed in. (It will be undefined if the user didn't supply one). That will help decide what we display in TAP.

`event.originalMessage` is provided for every assertion, including the "non-enhanced" ones.